WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -0,0 +1,124 @@
+import UIKit
+
+extension ReaderSortingOption {
+
+    var localizedDescription: String? {
+        // TODO: check if strings are localized
+        switch self {
+        case .date:
+            return NSLocalizedString("Recent", comment: "Sorting option description")
+        case .popularity:
+            return NSLocalizedString("Popular", comment: "Sorting option description")
+        case .noSorting:
+            return nil
+        }
+    }
+
+    var image: UIImage? {
+        switch self {
+        case .date:
+            // TODO: use proper icon
+            return .gridicon(.calendar)
+        case .popularity:
+            // TODO: use proper icon
+            return .gridicon(.lineGraph)
+        case .noSorting:
+            return nil
+        }
+    }
+}
+
+class ReaderSortingOptionButton: UIControl {
+    var sourceView: UIView {
+        return chevronView
+    }
+
+    private lazy var iconView: UIImageView = {
+        let view = UIImageView()
+        view.tintColor = UIColor(light: UIColor(hexString: "4D4D4D"), dark: UIColor(hexString: "BFBFBF"))
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private lazy var label: UILabel = {
+        let view = UILabel()
+        view.textColor = UIColor(light: .black, dark: .white)
+        view.font = WPFontManager.systemSemiBoldFont(ofSize: 15.0)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private lazy var chevronView: UIImageView = {
+        let view = UIImageView()
+        view.image = .gridicon(.chevronDown)
+        view.tintColor = UIColor(light: UIColor(hexString: "4D4D4D"), dark: UIColor(hexString: "BFBFBF"))
```

Comment:
can we group the "magic numbers" (e.g. `15.0`) together(e.g. in an `enum`) so they are easily identifiable and modifiable?